contrib/snyk/check-for-cve.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+PWD=/Users/user/.../hypershift/contrib/snyk
```

Comment:
You could just get the script directory
https://codefather.tech/blog/bash-get-script-directory/